filebeat/module/redis/log/ingest/pipeline.json

```diff
@@ -1,51 +1,81 @@
 {
-  "description": "Pipeline for parsing redis logs",
-  "processors": [
-    {
-      "grok": {
-        "field": "message",
-        "patterns": [
-          "(%{POSINT:process.pid:long}:%{CHAR:redis.log.role} )?%{REDISTIMESTAMP:redis.log.timestamp} %{REDISLEVEL:log.level} %{GREEDYDATA:message}",
-          "%{POSINT:process.pid:long}:signal-handler \\(%{POSINT:redis.log.timestamp}\\) %{GREEDYDATA:message}"
-        ],
-        "pattern_definitions": {
-          "CHAR": "[a-zA-Z]",
-          "REDISLEVEL": "[.\\-*#]"
+    "description": "Pipeline for parsing redis logs",
```

Comment:
I assume your editor applied here `.editorconfig` which is good: https://github.com/elastic/beats/blob/master/.editorconfig I wonder why this didn't happen for the nginx pipeline.